index.html

```diff
@@ -29,5 +29,111 @@ <h4>let's connect</h4>
+        </section>
+    
+        <section class="skills">
+            <img class="sample" src="./images/Snapshoot3.png" alt="project snapshoot">
+            <header class="project-header">
+                <h2>tonic</h2>
+                <ul class="tags">
+                    <li class="diff-li">canopy</li>
+                    <li>back end dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="set-stack">
+                    <li class="stack">html</li>
+                    <li class="stack">css</li>
+                    <li class="stack">javascipt</li>
+                </ul>
+                <button class="btn" type="submit">see project</button>
+            </header>
+        </section>
+    
+        <section class="skills">
+            <img class="sample" src="./images/Snapshoot4.png" alt="project snapshoot">
+            <header class="project-header">
+                <h2>tonic</h2>
+                <ul class="tags">
+                    <li class="diff-li">canopy</li>
+                    <li>back end dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="set-stack">
+                    <li class="stack">html</li>
+                    <li class="stack">css</li>
+                    <li class="stack">javascipt</li>
+                </ul>
+                <button class="btn" type="submit">see project</button>
+            </header>
+        </section>
+    </section>
+
+    <section class="about-myself banner">
+        <h2 class="intro">About<br> Myself!
+        </h2>
+        <p class="dev">
+            I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+        </p>
+        <h4>let's connect</h4>
+        <ul>
+            <li><a href="#"><img src="./images/twitter.svg" alt="twitter"></a></li>
+            <li><a href="#"><img src="./images/linkedin.svg" alt="linkedin"></a></li>
+            <li><a href="#"><img src="./images/Vector1.svg" alt=""></a></li>
+            <li><a href="#"><img src="./images/github.svg" alt="github"></a></li>
+            <li><a href="#"><img src="./images/Vector3.svg" alt=""></a></li>
+        </ul>
+        <button class="btn" type="submit">get my resume</button>
+        <ul class="pro-list">
+            <li class="pro-item one"><p>languages</p><img src="./images/stack-btn.png" alt="button"></li>
+            <li class="languages">
+                <img src="./images/javascript.png" alt="javascipt">
+                <img src="./images/html.png" alt="html">
+                <img src="./images/css.png" alt="css">
+            </li>
```

Comment:
Kindly wrap the list of languages in a `<ul>` tag 😃 and each language in a `<li>` tag since this is a list of languages.

Reply:
I did exactly this and my linter flagged it as an error @awais-amjed

Reply:
@awais-amjed @levy002 what should do now

Reply:
@Allenkeys did you try this? as I mentioned [above](https://github.com/Allenkeys/portfolio/pull/2#discussion_r950706158)

```
<li class="languages">
    <ul>
        <li><img src="./images/javascript.png" alt="javascipt"></li>
        <li><img src="./images/html.png" alt="html"></li>
        <li><img src="./images/css.png" alt="css"></li>
    </ul>
</li>
```

Reply:
@awais-amjed this is what I have now:

       <ul class="pro-list">
            <li class="pro-item one">
                <div>
                    <p>languages</p>
                    <img src="./images/stack-btn.png" alt="button">
                </div>
                <ul class="languages">
                    <li><img src="./images/javascript.png" alt="javascipt"></li>
                    <li><img src="./images/html.png" alt="html"></li>
                    <li><img src="./images/css.png" alt="css"></li>
                </ul>
            </li>
            <li class="pro-item"><p>frameworks</p><img src="./images/stack-btn.png" alt="button"></li>
            <li class="pro-item"><p>skills</p><img src="./images/stack-btn.png" alt="button"></li>
        </ul>

I hope it is fine?

Reply:
@awais-amjed Hope I'm good to go now. Its my final review?

Reply:
@Allenkeys I believe that is correct 👏👏

Reply:
Implemented 👍